web/src/beta/features/Editor/DataSourceManager/Asset/index.tsx

```diff
@@ -81,17 +80,14 @@ const Asset: React.FC<DataProps> = ({ sceneId, onSubmit, onClose }) => {
         </InputGroup>
         {sourceType == "url" && (
           <>
-            <InputGroup
-              label="File Format"
-              description="File format of the data source you want to add.">
-              <Select value={fileFormat} onChange={setFileFormat}>
-                {["GeoJSON", "KML", "CZML"].map(op => (
-                  <Option key={op} value={op} label={op}>
-                    {op}
-                  </Option>
-                ))}
-              </Select>
-            </InputGroup>
+            <SelectField
+              value={fileFormat}
+              options={["GeoJSON", "KML", "CZML"].map(v => ({ key: v, label: v }))}
```

Comment:
Since the options I think will be the same no matter the language they can be constants. Can you pull them out of the react component so they don't need to be re-rendered each time? 🙏🏻

Reply:
Done!